apps/obsidian/styles.css

```diff
@@ -1,22 +1,19 @@
-@tailwind base;
```

Comment:
This is probably fine for now, but we might need to prefix `@tailwind utilities` as well.  Or we precompile the CSS at build.  Or Shadow DOM.